webapp/channels/src/components/widgets/modals/components/save_changes_panel.tsx

```diff
@@ -11,31 +11,49 @@ import './save_changes_panel.scss';
 
 export type SaveChangesPanelState = 'editing' | 'saved' | 'error' | undefined;
 
+const CLOSE_TIMEOUT = 1200;
```

Comment:
wondering if this should be passed as a prop, so the consumer can decide how much time the success banner is shown depending on the scenario.

Reply:
could be a good idea, and this can be the default value.

Reply:
I don't think we should make it a prop unless we really need to. If we decide to change this, I really can't see a reason that we'd want to do it for one place and not others